<launch>
	<remap from="rccar_write" to="cmd_vel" />
	<node name="joy_node" pkg="joy" type="joy_node" />
	<node name="joy2twist" pkg="joy2twist" type="joy2twist" />
	<node name="tuw_rccar_node" pkg="tuw_rccar" type="tuw_rccar_node" />
</launch>
